web-app/src/app/tabs/motor/motor.component.html

```diff
@@ -86,11 +91,14 @@
 		</div>
 		<div class="container-column right-side">
 			<div class="subheader">
-				<span class="title-left">Key 1 Motor</span>
-	    			<span *ngIf="motor1.alive; else elseBlock" class="buttonCentering"><img class="circle" src="../../../assets/GreenCircle.png" alt="Red Circle" id="circle"></span>
-	    			<ng-template #elseBlock><img class="circle" src="../../../assets/RedCircle.png" alt="Red Circle" id="circle"></ng-template>
-				<span class="title-right">Alive</span>
-			</div>
+				<span class="title-left">Key Motor 1</span>
+				<div class="alive-group">
+	    			<span *ngIf="motor1.alive; else elseBlock">
+	    				<img class="circle" src="../../../assets/GreenCircle.png" alt="Red Circle" id="circle">
```

Comment:
Sorry Bill, I'm not sure if I changed this in another branch but just incase, can you change the alt to Green Circle